test/acceptance/features/accounting/replace_eu_fund_code_fdmr.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Dual Accounting
+
+    As a payment service
+    I want the EU fund code to be applied to the payment request for FDMR, if applicable
+    so that D365 can process the request with the correct fund code
+
+    - EU marketing year is the transition period from EU which ended 31 December 2020
+    - DOM10: 
```

Comment:
Description for this can be something like, payment request marketing year is post EU